starlette/routing.py

```diff
@@ -847,3 +869,8 @@ def decorator(func: typing.Callable) -> typing.Callable:
             return func
 
         return decorator
+
+    def __repr__(self) -> str:
+        routes_count = len(self.routes)
+        noun = "route" if routes_count == 1 else "routes"
+        return f"{self.__class__.__name__}({routes_count} {noun})"
```

Comment:
Why not the routes? 🤔

Reply:
There may be many routes, and printing all of them is not helpful. Just some brief info about their count, IMO, is enough.

Reply:
But then... Is it really helpful to have just the number of routes? :thinking: